<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirviroOfflineEdbDockWidgetBase</class>
 <widget class="QDockWidget" name="AirviroOfflineEdbDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airviro offline editor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>EDB</string>
       </attribute>
       <widget class="QLineEdit" name="create_db_lineedit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="create_db_browse_btn">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>390</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>367</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Create offline database</string>
        </property>
       </widget>
       <widget class="QTreeView" name="edb_treeview">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>291</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="connect_btn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="host_lineedit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>33</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>66</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
       <widget class="QPushButton" name="download_edb_btn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="upload_edb_btn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>550</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
       <widget class="QPushButton" name="open_edb_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>550</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="open_db_lineedit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>510</y>
          <width>251</width>
          <height>33</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="open_db_browse_btn">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>510</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>489</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Select offline database</string>
        </property>
       </widget>
       <widget class="QPushButton" name="create_edb_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Emissions</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
